website/Contribution/CONTRIBUTING.rst

```diff
@@ -959,6 +959,15 @@ Javascript
 * Never add minified Javascript libraries
 * Use UpperCamelCase for class declarations
 
+You can autofix your javascript code. Once eslint installed, run the following lines locally: 
+
+```
+git clone https://github.com/OCA/pylint-odoo
+cd pylint-odoo
```

Comment:
why this cd? better to put the path directly in the following command, isn't it?

Reply:
indeed. done.